components/apimgt/org.wso2.carbon.apimgt.rest.api.store.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/store/v1/impl/TenantsApiServiceImpl.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (c) 2019 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.apimgt.rest.api.store.v1.impl;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.wso2.carbon.apimgt.impl.utils.APIUtil;
+import org.wso2.carbon.apimgt.rest.api.store.v1.*;
+import org.wso2.carbon.apimgt.rest.api.store.v1.dto.*;
+import org.apache.cxf.jaxrs.ext.MessageContext;
+import org.wso2.carbon.apimgt.rest.api.util.RestApiConstants;
+import org.wso2.carbon.apimgt.rest.api.util.utils.RestApiUtil;
+import org.wso2.carbon.user.api.UserStoreException;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+import javax.ws.rs.core.Response;
+
+import static org.wso2.carbon.apimgt.impl.APIConstants.TENANT_STATE_ACTIVE;
+import static org.wso2.carbon.apimgt.impl.APIConstants.TENANT_STATE_INACTIVE;
+
+public class TenantsApiServiceImpl implements TenantsApiService {
```

Comment:
Let's add the class comments

Reply:
fixed https://github.com/wso2/carbon-apimgt/pull/6811/commits/a840167e827f482f100d25dadea5b4a46dc53be1